lib/components/artist/artworks/index.js

```diff
@@ -68,6 +70,7 @@ class Artworks extends React.Component {
           <SerifText style={styles.text}>{title}</SerifText> <SerifText style={[styles.text, styles.count]}>({count})</SerifText>
         </SerifText>
         <ArtworksGrid
+          artworks={this.props.for_sale_artworks}
```

Comment:
This line shouldn’t be included, it was my (incorrect) example of using an aliased field.